src/test/ui/internal/internal-unstable-const.rs

```diff
@@ -1,14 +0,0 @@
-// Don't allow unstable features in stable functions without `allow_internal_unstable`.
```

Comment:
This test sounds like we still want to keep it -- it is an important part of our const stability story.

You just need to use something other than `transmute` to test this. Maybe a floating-point operation?